code/common/subscriptions.q

```diff
@@ -62,7 +62,7 @@ replay0:{[tabs;realsubs;schemalist;logfilelist;filters]
   origupd:@[value;`..upd;{{[x;y]}}];
   // only use tables user has access to
   subtabs:realsubs[`subtabs];
-  if[count where nullschema:0=count each schemalist;
+  if[count where nullschema:0=count each schemalist; // change needed here 
```

Comment:
what is the change needed here and when is it being done, is it post master push or now?